service-api/pom.xml

```diff
@@ -20,5 +20,20 @@
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
```

Comment:
Why do you need this? I believe you don't need the whole web, but just some of dependencies from it.